initfiles/componentfiles/configschema/xsd/dafilesrv.xsd

```diff
@@ -1,29 +1,39 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <xs:schema
-    xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" attributeFormDefault="unqualified"
-    xmlns:hpcc="someuri">
+        xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" attributeFormDefault="unqualified"
+        xmlns:hpcc="someuri">
     <xs:include schemaLocation="types.xsd"/>
     <hpcc:insert hpcc:schemaPath="/Environment/Software">
-        <xs:element name="DafilesrvProcess" hpcc:itemType="dafilesrv" minOccurs="1" maxOccurs="1" hpcc:class="valueSet" hpcc:docid="daFs.t">
+        <xs:element name="DafilesrvProcess" hpcc:itemType="dafilesrv" minOccurs="1" maxOccurs="1" hpcc:class="valueSet"
+                    hpcc:docid="daFs.t">
             <xs:complexType>
                 <xs:attributeGroup name="executionSettings">
-                    <xs:attribute name="parallelRequestLimit" type="percent" use="optional" hpcc:presetValue="20" hpcc:tooltip="Defines the maximum number of concurrent dafilesrv requests allowed. Requests that exceed the limit will be delayed. A value of 0 disables throttling. Overrides global settting."/>
-                    <xs:attribute name="throttleDelayMs" type="xs:nonNegativeInteger" use="optional" hpcc:presetValue="5000" hpcc:tooltip="Defines how many milliseconds delayed requests will be delayed by. Overrides global settting."/>
-                    <xs:attribute name="throttleCPULimit" type="xs:nonNegativeInteger" use="optional" hpcc:presetValue="75" hpcc:tooltip="if after the initial delay, the CPU % falls below this setting, the transaction will be allowed to continue, i.e. the limit can be exceeded this way. Overrides global settting."/>
+                    <xs:attribute name="parallelRequestLimit" type="percent" use="optional" hpcc:presetValue="20"
```

Comment:
hpcc:presetValue: is it the default value?

Reply:
See the Wiki for a description of default values.